spec/event_builder_spec.rb

```diff
@@ -100,12 +110,10 @@
   end
 
   it 'should create a valid Event when create_impression_event is called with attributes as a string value' do
-    @expected_impression_params[:visitors][0][:attributes] = [{
-      entity_id: '111094',
-      key: 'browser_type',
-      type: 'custom',
-      value: 'firefox'
-    }]
+    @expected_impression_params[:visitors][0][:attributes].unshift(entity_id: '111094',
```

Comment:
why unshift? is there any issue in the existing line?

Reply:
Due to change in Order of visitor's attributes, tests failed. unshift pushes on first index.